app-misc/lcd4linux/lcd4linux-0.11.0_pre20170527.ebuild

```diff
@@ -0,0 +1,201 @@
+					net-wireless/wireless-tools ) )"
+
+RDEPEND="dmalloc? ( dev-libs/dmalloc )
+	${DEPEND_LCD_DEVICES}
+	${DEPEND_LCD4LINUX_PLUGINS}"
+
+DEPEND="${RDEPEND}"
+
+pkg_setup() {
+	# If USE="python", call python_setup
+	if use python; then
+		python-single-r1_pkg_setup
+	fi
+}
+
+src_prepare() {
+	# Install new version of ax_python_devel.m4, to fix support for >=dev-lang/python-3.
+	# Patch also plugin_python.c, to compile with >=dev-lang/python-3.
+	# The compilation seems also not to link against >=dev-lang/python-3 correctly, so adding it libs.
+	# See https://lcd4linux.bulix.org/wiki/plugin_python for more information.
+	if use python; then
+		if ! use python_single_target_python2_7; then
+			cp "${FILESDIR}"/ax_python_devel.m4 ax_python_devel.m4 || die
+			eapply "${FILESDIR}"/${PN}-0.11.0-add_python3_support.patch
+			append-libs "-lpython${EPYTHON#python}m"
+		fi
+	fi
+
+	# Apply user patches
+	eapply_user
+
+	# Run 'autotools'
+	eautoreconf
+}
+
+src_configure() {
+	# Define 'real names from configure' for 'LCD_DEVICES'
+	local LCD_DEVICES_NAMES=( ASTUSB BeckmannEgle BWCT CrystalFontz Curses Cwlinux D4D DPF
+					EA232graphic EFN FutabaVFD FW8888 G15 GLCD2USB HD44780 HD44780-I2C
+					IRLCD LCD2USB LEDMatrix LCDTerm LPH7508 LUIse
+					LW_ABP M50530 MatrixOrbital MatrixOrbitalGX MilfordInstruments MDM166A
+					Newhaven Noritake NULL Pertelian PHAnderson
+					PICGraphic picoLCD picoLCDGraphic PNG PPM RouterBoard
+					Sample SamsungSPF serdisplib ShuttleVFD SimpleLCD T6963
+					TeakLCM Trefon ULA200 USBHUB USBLCD VNC WincorNixdorf X11 )
+
+	# Enable all users enabled lcd devices
+	local myconf_lcd_devices
+	for lcd_device in ${!IUSE_LCD_DEVICES[@]}; do
+		if use lcd_devices_${IUSE_LCD_DEVICES[$lcd_device]}; then
+			if [ -z ${myconf_lcd_devices} ]; then
+				myconf_lcd_devices="${LCD_DEVICES_NAMES[$lcd_device]}"
+			else
+				myconf_lcd_devices="${myconf_lcd_devices},${LCD_DEVICES_NAMES[$lcd_device]}"
+			fi
+		fi
+	done
+
+	# Enable all users enabled lcd4linux plugins
+	local myconf_lcd4linux_plugins
+	for lcd4linux_plugin in ${!IUSE_LCD4LINUX_PLUGINS[@]}; do
+		if use ${IUSE_LCD4LINUX_PLUGINS[$lcd4linux_plugin]}; then
+			if [ -z ${myconf_lcd4linux_plugins} ]; then
+				myconf_lcd4linux_plugins="${IUSE_LCD4LINUX_PLUGINS[$lcd4linux_plugin]}"
+			else
+				myconf_lcd4linux_plugins="${myconf_lcd4linux_plugins},${IUSE_LCD4LINUX_PLUGINS[$lcd4linux_plugin]}"
+			fi
+		fi
+	done
+
+	# Run 'configure'
+	econf \
+		--disable-rpath \
+		$(use_with dmalloc) \
+		$(use_with outb) \
+		$(usex python "--with-python PYTHON_VERSION=${EPYTHON#python}" "--without-python") \
+		$(usex lcd_devices_x11 "--with-x" "--without-x") \
+		$(usex lcd_devices_x11 "--x-include=/usr/include --x-libraries=/usr/$(get_libdir)" "") \
+		--with-drivers="${myconf_lcd_devices}" \
+		--with-plugins="${myconf_lcd4linux_plugins}"
+}
+
+src_install() {
+	# Set docs
+	local DOCS=( "AUTHORS" "ChangeLog" "CREDITS" "INSTALL" "NEWS" "README" "TODO" )
```

Comment:
You can probably remove that line, EAPI6 `default` should install relevant files by itself.

Reply:
Yes, but without my own DOCS, not all of that specified documents are installed.

Reply:
okay then.

Reply:
Thanks, so I don't changed this.

Reply:
The only document that won't be installed is INSTALL... and that makes sense because people don't need to know how to install the package using that instructions (they rely on the ebuild for that). I would then simply drop the line

Reply:
Allright ;-) Dropped it and pushed a new version.